app/routers/books.py

```diff
@@ -20,32 +25,40 @@
 # Local application imports
 # -------------------------
 from config import settings
-
 from ..applogger import rslogger
 from ..crud import create_useinfo_entry
 from ..db import database as db
 from ..schemas import LogItem, LogItemIncoming
 
-#
+# Routing
+# =======
 # Setup the router object for the endpoints defined in this file.  These will
-# be connected to the main application in main.py
-#
+# be `connected <included routing>` to the main application in `../main.py`.
 router = APIRouter(
-    prefix="/books",  # shortcut so we don't have to repeat this part
-    tags=["books"],  # groups all logger tags together in the docs
+    # shortcut so we don't have to repeat this part
```

Comment:
Is there any other option for including these comments?  I find this style to be very distracting.  To me it makes the function call a lot harder to read and understand.  Also, stylistically we should decide, do we need to repeat this in every router file?  I notice that you left rslogger.py alone.  I know someone could dive in to any given router file, so it doesn't make sense to just do it in one file.  But too much repetition is not good either.

Reply:
Good questions. The CodeChat system can't support mixed comments and code -- [here's](https://codechat.readthedocs.io/en/master/CodeChat/CodeToRest.py.html#mixed-code-and-comments) a (somewhat obscure) explanation why. So, when using CodeChat, avoid mixed comments and code.

Given that, I'm open to other styles -- [here](https://codechat.readthedocs.io/en/master/docs/style_guide.cpp.html#an-unindented-heading-correct) is my style guide and reasoning for the style I use. What would make more sense to you?

Reply:
To repeat something, we can use reST replacement syntax. Another (perhaps better) option: document it in one place, then hyperlink to that from other similar code.

I'll do that, then tell me what you think.